frontend/src/components/layout/Sidebar.tsx

```diff
@@ -0,0 +1,179 @@
+              onClick={() => toggleSection('courses')}
+              aria-label={t('sidebar.toggle_courses')}
+          >
+            <div className="flex items-center space-x-2">
+              <FolderClosed size={20} />
+              {!isCollapsed && <span>{t('sidebar.my_courses')}</span>}
+            </div>
+            {!isCollapsed && (expandedSections.courses ? <ChevronDown size={16} /> : <ChevronRight size={16} />)}
+          </button>
+          {/* Favorite Courses List */}
+          {!isCollapsed && expandedSections.courses && (
+
+              <div>
+                <ItemList
+                    items={mapCoursesToItems(user!.favoriteCourses!)}
+                    emptyMessage={t('account.favorite.no_courses')}
+                    updateFavorite={updateFavoriteCourse}
+                />
+              </div>
+
+          )}
+          <button
+              className="flex items-center justify-between w-full text-gray-700 hover:bg-gray-100 transition-all duration-200 hover:scale-[1.01] rounded p-2"
+              onClick={() => toggleSection('documents')}
+              aria-label={t('sidebar.toggle_documents')}
+          >
+            <div className="flex items-center space-x-2">
+              <File size={20} />
+              {!isCollapsed && <span>{t('sidebar.my_documents')}</span>}
+            </div>
+            {!isCollapsed && (expandedSections.documents ? <ChevronDown size={16} /> : <ChevronRight size={16} />)}
+
+          </button>
+          {/* Favorite Documents List */}
+          {!isCollapsed && expandedSections.documents && (
+              <div>
+                <ItemList
+                    items={mapDocumentsToItems(user!.favoriteDocuments!)}
+                    emptyMessage={t('account.favorite.no_documents')}
+                    updateFavorite={updateFavoriteDocument}
+                />
+              </div>
+          )}
+        </nav>
+
+        {/* Add Document Button */}
+        <button
+
+            className={`mx-4 my-2 flex items-center space-x-2 ${isCollapsed ? 'bg-transparent' : 'bg-indigo-600 hover:bg-indigo-700 transition-all duration-200 hover:scale-[1.01]'} text-white rounded-md py-2 px-4 hover:${isCollapsed ? 'bg-transparent' : 'bg-indigo-700'}`}
+            aria-label={t('sidebar.add_document')}
+            onClick={handleUploadButtonClick}
+        >
+          <Plus size={20} />
+          {!isCollapsed && <span>{t('sidebar.add_document')}</span>}
+        </button>
+
+
+        {/* User Profile - Fixed at Bottom #TODO add avatar input*/}
+        <div className="mt-auto border-t border-gray-200">
+          <div className="p-4 flex items-center space-x-3">
+            <img
+                src={'/images/icons/empty_profile.png'}
+                alt={t('sidebar.user_avatar')}
+                className="w-8 h-8 rounded-full"
+            />
```

Comment:
./src/components/layout/Sidebar.tsx
148:13  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
```suggestion
            <Image
                src={'/images/icons/empty_profile.png'}
                alt={t('sidebar.user_avatar')}
                className="w-8 h-8 rounded-full"
            />
```